src/librustc_infer/infer/lexical_region_resolve/graphviz.rs

```diff
@@ -1,5 +1,5 @@
 //! This module provides linkage between libgraphviz traits and
-//! `rustc::middle::typeck::infer::region_constraints`, generating a
+//! `rustc_middle::middle::typeck::infer::region_constraints`, generating a
```

Comment:
Does this module even exist anymore, it looks off.

Reply:
Yeah I don't think it does.